wiki/markdown/curl-examples.md

```diff
@@ -1,40 +1,110 @@
-* Default back-end
-* Input back-end
+More information and examples can be found in the [EI back-end documentation](https://github.com/eiffel-community/eiffel-intelligence/blob/master/wiki/markdown/Authentication.md)
+
+
+## <a id="backend" /> /backend
+
+<table>
+    <tr>
+        <th>Context Path</th>
+        <th>Type</th>
+        <th>Explanation</th>
+    </tr>
+    <tr>
+        <td>/backend</td>
+        <td>GET</td>
+        <td>Retrieves a json list of available back-end instances</td>
+    </tr>
+    <tr>
+        <td>/backend</td>
+        <td>POST</td>
+        <td>Adds one (or more) back-end instance(s)</td>
+    </tr>
+</table>
 
 While the web GUI may use the back-end instances list and select different back-ends in an easy way, a user that uses the front-end without the included web GUI may need to specify a back-end URL. This may be done by adding a `backendurl` parameter to the request.
 
 Note that for users where the front-end and back-end is running with the help of Tomcat there will be context paths used.
 
-### Default back-end
+#### Default back-end
 
-The front-end may have been setup with a default back-end, this means that if no back-end is specified when making a HTTP request to the front-end the default back-end will be used.
-If you want to see a list of back-ends and see if there is a default back-end set you may use the command:
+The front-end can be configured to use a default back-end, this means that if no back-end is specified when making a HTTP request to the front-end the default back-end will be used.
+If you want to see a list of back-ends and see if there is a default back-end set, you may use the command:
 
-###### Normal curl command for get:
+###### Example GET request using curl:
 
-    curl -X GET http://*front-end-url*/*contect-path-if-any*/backend
+    curl -X GET http://*front-end-url*/*context-path-if-any*/backend
 
-###### Curl command for get with host, port using Tomcat:
+###### Example GET request using curl, with host, port using Tomcat:
 
     curl -X GET http://localhost:8080/eifrontend/backend
 
-###### Curl command using EI default settings:
+###### Example GET request using curl, with EI default settings:
 
     curl -X GET http://localhost:8080/backend
 
 The default back-end should have the key `defaultBackend` set to `true`. If the JSON list ends up empty there are no back-ends specified in the front-end. If there is no JSON object with the key set to true there is no default back-end.
 
-You may see that in the list, all objects have a key `active` set to `true or false`, this key points to the back-end that acts as default, if no default back-end exist, the active should usually be the first object in the list.
+In the response list all objects have a key `active` set to `true` or `false`. This key reveals which back-end instance has been chosen as default in the GUI. If no default back-end exist, the active should usually be the first object in the list.
```

Comment:
This sentence need to be changed, active indicates what instance would be selected in the GUI, a curl command will always display the default as active, or the first element in the list if no default exist.

Reply:
Fixed.